models/gif_pop_psc_exp.cpp

```diff
@@ -526,22 +527,18 @@ nest::gif_pop_psc_exp::update( Time const& origin,
     double JNy_ex_ = S_.I_syn_ex_ / P_.c_m_;
     double JNy_in_ = S_.I_syn_in_ / P_.c_m_;
 
-    // membrane update (line 10)
-    double h_ex_ =
-      P_.tau_m_
-      * ( JNA_ex_
-          + ( P_.tau_syn_ex_ * V_.P11_ex_ * ( JNy_ex_ - JNA_ex_ )
-              - V_.P22_ * ( P_.tau_syn_ex_ * JNy_ex_ - P_.tau_m_ * JNA_ex_ ) )
-            / ( P_.tau_syn_ex_ - P_.tau_m_ ) );
-    double h_in_ =
-      P_.tau_m_
-      * ( JNA_in_
-          + ( P_.tau_syn_in_ * V_.P11_in_ * ( JNy_in_ - JNA_in_ )
-              - V_.P22_ * ( P_.tau_syn_in_ * JNy_in_ - P_.tau_m_ * JNA_in_ ) )
-            / ( P_.tau_syn_in_ - P_.tau_m_ ) );
+    // membrane update (line 10 of [1])
+    double h_ex_tmpvar_ = ( P_.tau_syn_ex_ * V_.P11_ex_ * ( JNy_ex_ - JNA_ex_ )
+              - V_.P22_ * ( P_.tau_syn_ex_ * JNy_ex_ - P_.tau_m_ * JNA_ex_ ) );
+    double h_in_tmpvar_ = ( P_.tau_syn_in_ * V_.P11_in_ * ( JNy_in_ - JNA_in_ )
+              - V_.P22_ * ( P_.tau_syn_in_ * JNy_in_ - P_.tau_m_ * JNA_in_ ) );
+    double h_ex_ = P_.tau_m_ * ( JNA_ex_ + 
+                    h_ex_tmpvar_ / ( P_.tau_syn_ex_ - P_.tau_m_ ) );
+    double h_in_ = P_.tau_m_ * ( JNA_in_ + 
+                    h_in_tmpvar_ / ( P_.tau_syn_in_ - P_.tau_m_ ) );
```

Comment:
The compiler may be able to optimize better if you declare these four variables `const`. Also, local variable names should not end with `_`, that suffix is only for private members.